hack/commits/README

```diff
@@ -0,0 +1,6 @@
+This directory includes files with a commit hash for 
```

Comment:
I was expect you to simply tag origin-metrics and origin-logging, then I would do `git checkout v1.1.6` and run `OS_TAG=v1.1.6 hack/release.sh` from your repo.